packages/extension-api/src/extension-api.d.ts

```diff
@@ -202,9 +202,15 @@ declare module '@tmpwip/extension-api' {
     create(params: { [key: string]: any }): Promise<void>;
   }
 
+  export interface Link {
+    name: string;
+    url: string;
+  }
+
   export interface CheckResult {
     successful: boolean;
     description?: string;
+    docLinks?: Link[];
```

Comment:
We have a ProviderLink class with title and url in the API

maybe we should use CheckResultLink for now extending Link and ProviderLink extending Link as well

Reply:
TS compiler will complain if `CheckResultLink` and `ProviderLink` not add own fields/methods.

Reply:
what do you suggest ? rename ProviderLink to Link ?

Reply:
I done that with `type` instead of interface